tests/node/.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+    "parserOptions": {
+        "ecmaVersion": 6
```

Comment:
Should this be 6 or 5?

Reply:
6 for template strings used in the tests.